.github/renovate.json

```diff
@@ -20,11 +20,12 @@
         "^@backstage/",
         "^@janus-idp/",
         "^@immobiliarelabs/",
-        "^@roadiehq/"
+        "^@roadiehq/",
+        "^@pagerduty/"
       ],
       "groupName": ["Backstage packages"],
       "dependencyDashboardApproval": true
     }
   ],
-  "ignorePaths": ["**/dist-dynamic/**"]
+  "ignorePaths": ["**/dynamic-plugins/**"]
```

Comment:
Why this change ?

Reply:
The plugin folder will be updated manually through the backstage `versions:bump` cli. Currently renovate is creating PRs for dependencies we cannot update in `dynamic-plugins`.

Reply:
> dependencies we cannot update in dynamic-plugins.

Can you precise which ones ?

Reply:
Hmmm, thinking about it more I guess we don't need the change. Reverting...